flume-ng-sinks/flume-http-sink/src/main/java/org/apache/flume/sink/http/HttpSink.java

```diff
@@ -27,7 +27,8 @@
 import org.apache.flume.conf.Configurable;
 import org.apache.flume.instrumentation.SinkCounter;
 import org.apache.flume.sink.AbstractSink;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
```

Comment:
I would prefer the slf4j logger here because that way the user could decide to use a different logger implementation.
(despite the fact that the original code already used log4j)

Reply:
Because it was using log4j I converted it to use log4j...

Reply:
Flume components are using slf4j. I guess this log4j logger in the HttpSink was overlooked at the review and that is why it is not an slf4j logger. It would be nice if it could be consistent.

Reply:
I have no problem if that change is made but it doesn't necessarily have to happen as part of this PR.

Reply:
I agree. I will make a different PR about it. Thank you.